.circleci/config.yml

```diff
@@ -127,18 +120,42 @@ workflows:
               ignore:
                 - master
 
-      - test:
+      # UNIX tests
+      - test-unix:
+          name: Unix Tests for Node << matrix.node_version >>
+          context: nodejs-install
+          requires:
+            - lint
+          matrix:
+            parameters:
+              node_version: ["14.17.6", "12.0.0", "10.21.0"]
+          filters:
+            branches:
+              ignore:
+                - master
+
+      # Windows tests
+      - test-windows:
+          name: Windows Tests for Node << matrix.node_version >>
+          context: nodejs-install
+          requires:
+            - lint
           matrix:
             parameters:
-              os: [linux, macos, windows]
-              node_version: ["10.22", "12.18", "14.8"]
+              node_version: ["14.17.6", "12.0.0", "10.21.0"]
+          filters:
+            branches:
+              ignore:
+                - master
 
       - release:
           name: Release
           context: nodejs-lib-release
+          node_version: "lts"
```

Comment:
Great work here @ekbsnyk! 🎉 
I would like just to add a small suggestion here, that would be to change `node_version` release from `lts` to node v10 which is the min supported version by our cli tool rather than the latest node released by node